kermit-saas/private.tf

```diff
@@ -303,4 +303,31 @@ resource "aws_instance" "inst_kermit_build_u16" {
 
 output "inst_kermit_build_u16_priv_ip" {
   value = "${aws_instance.inst_kermit_build_u16.private_ip}"
-}
\ No newline at end of file
+}
+
+## RHEL7 Instances
+resource "aws_instance" "inst_kermit_rhel7" {
+ ami = "${var.ami_us_east_1_rhel7}"
+ availability_zone = "${var.avl_zone}"
+ instance_type = "${var.inst_type_x}"
+ key_name = "${var.aws_key_name}"
+ subnet_id = "${aws_subnet.sn_private_kermit.id}"
+
+ count = 2
+ vpc_security_group_ids = [
+   "${aws_security_group.sg_private_kermit.id}"]
+
+ root_block_device {
+   volume_type = "gp2"
+   volume_size = 100
```

Comment:
Do you really need a disk this big to test with?

Reply:
No, i will change that size.